Packages/macOS/CmuxTerminal/Tests/GhosttyRuntimeTestStubs/GhosttyRuntimeTestStubs.c

```
@@ -361,7 +471,46 @@ bool ghostty_surface_process_exited(void *surface) {
 }
 void ghostty_surface_process_output(void) {}
 void ghostty_surface_quicklook_font(void) {}
-void ghostty_surface_read_screen_tail_vt(void) {}
+bool ghostty_surface_read_screen_tail_vt(
```

Cppcheck (2.21.0)

[style] The function 'ghostty_surface_read_screen_tail_vt' is never used.

(unusedFunction)